app/src/main/res/drawable/ic_lock_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Could you please say where you got this icons and the other ones from?

Reply:
All 4 icons are from [material.io](http://material.io)